saltproc/app.py

```diff
@@ -30,22 +34,22 @@ def run():
     simulation.check_restart()
     # Run sequence
     # Start sequence
-    for dep_step in range(len(msr.dep_step_length_cumulative)):
-        print("\n\n\nStep #%i has been started" % (dep_step + 1))
+    for step_idx in range(len(msr.depletion_timesteps)):
```

Comment:
Is there a difference between the step index and the step number? I would just call it ``step``. Though this is purely a stylistic difference.

Reply:
I'm just trying to be as explicit as possible.

Reply:
Okay, I don't see a difference but it doesn't hurt.